docs/wip_glassbox/wip-glassbox-customization.md

```diff
@@ -3,7 +3,10 @@
 ## Entities
 * visibility of all private properties and methods of DataFactories was changed to protected (@Miroslav-Stopka)
     * protected visibility allows overriding of behavior from projects
-    
+* visibility of all private properties and methods of repositories of entities was changed to protected (@Miroslav-Stopka)
```

Comment:
Please move this point also to the bottom Entities section

Reply:
Done